kolibri/core/assets/src/state/modules/core/actions.js

```diff
@@ -243,8 +243,14 @@ export function kolibriLogin(store, sessionPayload) {
   Lockr.set(UPDATE_MODAL_DISMISSED, false);
   return SessionResource.saveModel({ data: sessionPayload })
     .then(() => {
-      // Redirect on login
-      redirectBrowser();
+      // OIDC redirect
+      if (sessionPayload.next) {
+        redirectBrowser(sessionPayload.next);
```

Comment:
This is where redirect can occur. Unsure if this is actually what's supposed to happen next in the flow?